### DISCUSSION TITLE
Question about Pool switching

### DISCUSSION THREAD

**Discussion by yarnauy** [Chia-Network/chia-blockchain#1713] [Q&A]
Does Chia support pool switching without replotting? 
It seems that I am still required to give my secret key to the pool? or can I only give a signature to the pool and invalidate the signature when I want to switch to another pool?

**Comment by dorkmo** [comment]
dont think you want to give your private key to anyone

**Reply by yarnauy** [reply]
How can pools get the rewards earned by my plots file without the key?

**Comment by hoffmang9** [comment]
https://www.chia.net/2020/11/10/pools-in-chia.html

**Reply by yarnauy** [reply]
I've just read this, but I don't know if Chia has complete the pool protocol in this article.
